site/content/en/references/kustomize/builtins/_index.md

```diff
@@ -757,25 +757,17 @@ helmChartInflationGenerator:
 
 #### Arguments
 
-> ChartName string
+> Name string
```

Comment:
Lines 730-753 need to be updated accordingly as well to reflect the updated fields. For example, extraArgs is no longer supported. You can find the new fields in the source code here https://github.com/kubernetes-sigs/kustomize/blob/416eed97c49aa8e9ea44d627c15dafd395b61739/api/types/helmchartargs.go#L6-L75

Reply:
@natasha41575 thanks for the review, please see the changes